apps/pom.xml

```diff
@@ -37,38 +37,32 @@
     <dependency>
       <groupId>${edgent.runtime.groupId}</groupId>
       <artifactId>edgent-analytics-sensors</artifactId>
-      <version>${edgent.runtime.version}</version>
     </dependency>
     <dependency>
       <groupId>${edgent.runtime.groupId}</groupId>
       <artifactId>edgent-connectors-file</artifactId>
-      <version>${edgent.runtime.version}</version>
     </dependency>
     <dependency>
       <groupId>${edgent.runtime.groupId}</groupId>
       <artifactId>edgent-analytics-math3</artifactId>
-      <version>${edgent.runtime.version}</version>
     </dependency>
     <dependency>
       <groupId>${edgent.runtime.groupId}</groupId>
       <artifactId>edgent-connectors-iot</artifactId>
-      <version>${edgent.runtime.version}</version>
     </dependency>
     <dependency>
       <groupId>${edgent.runtime.groupId}</groupId>
       <artifactId>edgent-connectors-iotp</artifactId>
-      <version>${edgent.runtime.version}</version>
     </dependency>
     <dependency>
       <groupId>${edgent.runtime.groupId}</groupId>
       <artifactId>edgent-connectors-mqtt</artifactId>
-      <version>${edgent.runtime.version}</version>
     </dependency>
     
     <dependency>
-      <groupId>${project.groupId}</groupId>
+      <groupId>org.apache.edgent</groupId>
```

Comment:
Woops, I was wrong about this: This groupId change will break building the samples for java7.
But, I think it should be reverted to its original value (same rationale as for version below).

Also, I think the following change to version is neither necessary nor desirable (since project.version is a static value defined in the pom itself).  I'll note there are similar occurrences in other poms (connectors, scenarios,topology) that weren't changed and those apparently didn't cause a problem.

Reply:
Yeah ... well the same problem applies to the groupId as for the version ... unfortunately I can't move the groupId to dependency management as I could the version. So I guess duplicating the properties in every pom seems to be the only option.

Reply:
But I also think: In contrast to the Edgent libs themselves it would be safe to fix the samples against one platform. After all they are examples no production-ready solutions.